.github/workflows/rules.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
@stavros11 @mlazzarin if I remember correctly, the problem with 3.6 is due to the matplotlib dependency.
I don't think qibo should depend on matplotlib, we can include a special installation target ['examples'] which install matplotlib, what do you think?

Reply:
I think the py36 issue is solved if we install qibo from source but using `pip install .`. I am not sure what was causing this issue, the error was indeed related to matplotlib, however it is not appearing in other PRs for example in qibo repository.

Reply:
Thanks for this, very good that it works.